src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2270,182 +2270,4 @@ begin
     rw [nat.cast_zero, opow_zero] }
 end
 
-/-! ### Fixed points of normal functions -/
-
-section
-variable {f : ordinal.{u} → ordinal.{u}}
-
-/-- The next fixed point function, the least fixed point of the normal function `f` above `a`. -/
-def nfp (f : ordinal → ordinal) (a : ordinal) :=
-sup (λ n : ℕ, f^[n] a)
```

Comment:
Can you do this move in a separate PR in order to make the diff easier to follow?

Reply:
Sure, I'll split this up once the current merge issues are resolved.

Reply:
Done! I'll reintroduce this move in a follow-up PR.